data/init_data.csv

```diff
@@ -0,0 +1,1001 @@
+star_no,first_name,last_name,job_title,department_id
+1006,Mariel,Boullen,Editor,1
+1012,Ransom,Million,Speech Pathologist,1
```

Comment:
You know what they say "ACAB but especially those speech pathologists!"

Reply:
Hehehe look it's what [Mockaroo](https://www.mockaroo.com/) was supplying for job titles 🤷🏼‍♀️ 😁